data/strings/types_strings.txt

```diff
@@ -5771,7 +5615,7 @@
     zh-Hant = 木工
 
   [type.craft.confectionery]
-    en = Confectionery
+    en = Confectioner
```

Comment:
In en_US, that's "candy shop".

Reply:
candy shop is shop-confectionery, craft-confectionery is for artisan/handmade etc. added Candy Shop to shop-confectionery.